samples/Samples.AspNetCoreMvc31/Properties/launchSettings.json

```diff
@@ -39,7 +39,12 @@
         "OTEL_DOTNET_TRACER_HOME": "$(ProjectDir)$(OutputPath)profiler-lib",
         "OTEL_INTEGRATIONS": "$(ProjectDir)$(OutputPath)profiler-lib\\integrations.json",
         "OTEL_VERSION": "1.0.0",
-        "OTEL_TRACE_HEADER_TAGS": "sample.correlation.identifier, Server"
+        "OTEL_SERVICE": "Samples.AspNetCoreMvc31",
+        "OTEL_TRACE_HEADER_TAGS": "sample.correlation.identifier, Server",
+
+        "OTEL_EXPORTER": "jaeger",
```

Comment:
Will this really work on the POC branch? I mean are we copying it under the sample now?

Reply:
Yes, launch profiles do work if you run these directly. For testing purposes I can see that no launch profiles are used (there is a flag --no-launch-profile).